src/lib/y2network/dialogs/edit_interface.rb

```diff
@@ -47,10 +48,12 @@ def contents
         VBox(CWM::Tabs.new(*tabs))
       end
 
+      # abort is just cancel as this is a sub dialog
       def abort_button
         Yast::Label.CancelButton
       end
 
+      # removes back button when editing device, but keep it when this dialog followed add
       def back_button
```

Comment:
missing docu ... can be important as the method is not straightforward

Reply:
I do not get what you mean here with missing documentation? something like `see CWM::Dialogs#back_button`  ?

Reply:
hm not ... seems I've overlooked something. Forget it.